### DISCUSSION TITLE
Running the jar file in a python script?

### DISCUSSION THREAD

**Discussion by mattcutini** [synthetichealth/synthea#1202] [Q&A] [answered]
Is there a recommended way to run the synthea jar file within a python script? I'm trying to use subprocess.call() but it appears to not accept the flags that I'm trying to add.

**Comment by W3bParserV2** [comment, marked as answer]
### Have you tried the os.system package ?

**Try to run it by the following way? This is what i mean :**
At the start of your python script write this : 

```python
import os
```

And when you want to run your jar file, call _os.system_ like this :

```python
os.system('cmd /c "jar -jar file.jar"')
```

Try it and send me the output if it doesn't work.

If you want just open the jar file and not run it, do this : 

```python
os.system('cmd /c "jar xf file.jar"')
```

**Reply by mattcutini** [reply]
I was able to get this to run:

`os.system("java -jar synthea-with-dependencies.jar -p 100")`

Thank you for your help!

**Reply by W3bParserV2** [reply]
np
